.github/workflows/func.yaml

```diff
@@ -136,7 +141,7 @@ jobs:
         run: |
           sudo apt update -yqq
           sudo apt remove -qy lxd
-          if [[ $(cat /etc/os-release | grep VERSION_CODENAME) == *"focal"* ]]; then \
+          if [[ $(cat /etc/os-release | grep VERSION_CODENAME) == *"focal"* ]]; then
             suod apt remove -qy lxd-client
```

Comment:
```suggestion
            sudo apt-get remove -q=2 lxd-client
```

Reply:
I notice this is not part of your commit, how does it works before?

Reply:
regression from #41

Reply:
I don't understand why you suggest this change. Can you provide some comment?

Reply:
typo: `suod`

Reply:
Ok, I think the reason it's pass it's because the runner is jammy.

Reply:
Updated.